backend/src/api/transactions/transaction.model.js

```diff
@@ -0,0 +1,31 @@
+const mongoose = require('mongoose');
+const {
+  Schema,
+  Types: { ObjectId },
+} = mongoose;
+const { transactionCategories } = require('../dataset');
+
+const transactionSchema = new Schema({
+  amount: { type: Number, required: true },
+  transactionDate: { type: Date, required: true },
+  type: {
+    type: String,
+    enum: ['INCOME', 'EXPENSE'],
+    default: 'EXPENSE',
+    required: true,
+  },
+  comment: { type: String, default: '' },
+  category: {
+    type: Object,
+    enum: transactionCategories,
+    default: transactionCategories[0],
+  },
+  userId: { type: ObjectId, ref: 'User' },
+  createdAt: {
```

Comment:
The same stuff with `{ timestamps: true }` as I've mentioned in User model PR

Reply:
Second parameter 'timestamps' is added to this schema.